### ISSUE TITLE
OpenAPI allows additional fields at the level where methods are

### ISSUE THREAD

**Issue by Stwissel** [jormaechea/open-api-mocker#8]
I'm using an [Apicurio](https://apicur.io) edited OpenAPI 3.0x specification. When trying to load it, I get the error:

```
> open-api-mocker -s Playground.json
[FATAL] Error while validating schema: Expected a value of type `"get" | "post" | "put" | "patch" | "delete" | "options" | "head" | "trace"` for `/.description` but received `"description"`. in paths./.description
```

Hoever when looking at the specs, it is compliant. At the same level where the HTTP Verbs (get, post, put etc.) live, one can specify: `summary`, `description` or parameters

Example
```json
{
    "summary": "Path used to manage a single AutomationDefinition.",
    "description": "The REST endpoint/path used to get, update, and delete single instances of an `AutomationDefinition`.  This path contains `GET`, `PUT`, and `DELETE` operations used to perform the get, update, and delete tasks, respectively.",
    "get": {
        "tags": [
            "Operations"
        ],
        "responses": {
            "200": {
                "content": {
                    "application/json": {
                        "schema": {
                            "$ref": "#/components/schemas/AutomationDefinition"
                        }
                    }
                },
                "description": "Successful response - returns a single `AutomationDefinition`."
            }
        },
        "operationId": "getAutomationDefinition",
        "summary": "Get a AutomationDefinition",
        "description": "Gets the details of a single instance of a `AutomationDefinition`."
    },
    "put": {
        "requestBody": {
            "description": "Updated `AutomationDefinition` information.",
            "content": {
                "application/json": {
                    "schema": {
                        "$ref": "#/components/schemas/AutomationDefinition"
                    }
                }
            },
            "required": true
        },
        "responses": {
            "202": {
                "description": "Successful response."
            }
        },
        "operationId": "updateAutomationDefinition",
        "summary": "Update a AutomationDefinition",
        "description": "Updates an existing `AutomationDefinition`."
    },
    "delete": {
        "responses": {
            "204": {
                "description": "Successful response."
            }
        },
        "operationId": "deleteAutomationDefinition",
        "summary": "Delete a AutomationDefinition",
        "description": "Deletes an existing `AutomationDefinition`."
    },
    "parameters": [
        {
            "name": "automationdefinitionId",
            "description": "A unique identifier for a `AutomationDefinition`.",
            "schema": {
                "type": "string"
            },
            "in": "path",
            "required": true
        }
    ]
}
```

**Comment by jormaechea** [comment]
@Stwissel you're right! It's not validating the `summary` property. I'll try to fix it soon.

I leave the docs here for reference: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#pathItemObject

Thanks for the report!

**Comment by jormaechea** [comment]
@Stwissel well, that was fast..

This is fixed in v1.4.0 :rocket:
